feature/signupflow/entercode/src/main/java/com/epmedu/animeal/signup/entercode/domain/confirmcode/MobileConfirmCodeUseCase.kt

```diff
@@ -1,11 +1,12 @@
-package com.epmedu.animeal.signup.entercode.domain
+package com.epmedu.animeal.signup.entercode.domain.confirmcode
 
 import com.amplifyframework.auth.result.AuthSignInResult
 import com.epmedu.animeal.auth.AuthRequestHandler
 import com.epmedu.animeal.signup.entercode.data.EnterCodeRepository
 
-class ConfirmCodeUseCase(private val repository: EnterCodeRepository) {
-    operator fun invoke(
+class MobileConfirmCodeUseCase(private val repository: EnterCodeRepository) : ConfirmCodeUseCase {
```

Comment:
I hoped to use Hilt Qualifiers and create 2 different implementations of EnterCodeRepository, so those UseCases would not be duplicated for Mobile and Facebook purposes, but sadly, MobileConfirmCodeUseCase's requestHandler differs from Facebook's one - it acts accordingly to recieved AuthSignInResults, while in FacebookConfirmCodeUseCase the "result" argument inside of "onSuccess" returned as null.